lib/WasmReader/WasmSection.h

```diff
@@ -27,8 +27,8 @@ namespace Wasm
     {
         SectionFlag flag;
         SectionCode precedent;
-        char16* name;
-        char* id;
+        const wchar_t* name;
```

Comment:
+1. Please try avoid wchar_t.

---
In reply to: [83068286](https://github.com/Microsoft/ChakraCore/pull/1727#discussion_r83068286) [](ancestors = 83068286)